### DISCUSSION TITLE
universal ziki for android and any plataform

### DISCUSSION THREAD

**Discussion by greathector7** [zim-desktop-wiki/zim-desktop-wiki#2463] [Q&A] [answered]
Greetings, I have been using the tool for years, with the advantage of using it in Windows and Linux and although there is still no port for Android-Ios, it seems to me that the leap can be made since I use the tool together with dropbox and that allows me have everything synchronized with a single zim note book. It would only be porting the code to a client who can read that same book on the phone.

**Comment by dehesselle** [comment, marked as answer]
Have you seen https://github.com/zim-desktop-wiki/zim-desktop-wiki/discussions/1737#discussioncomment-1613477?
